src/pages/open-source.js

```diff
@@ -12,6 +13,11 @@ const OpenSource = ({ data }) => {
       slug,
       seoTitle,
       seoMetaDescription,
+      talksSectionImage,
```

Comment:
We could group these as @omonk is doing in his PR & then pass in `{...data}` to the `TalksSection`. What do you think @GBouffard @omonk ?

For example, the contributions section in `open-source.js`:
https://github.com/yldio/yld.io/blob/af1490401b5ef88e7bcfe7256987e2637945e412/src/pages/open-source.js#L12-L29

and then destructuring within `Contributions.js`:
https://github.com/yldio/yld.io/blob/af1490401b5ef88e7bcfe7256987e2637945e412/src/components/OpenSource/Contributions.js#L28-L48

Reply:
By passing `...data` you also pass a lot of things that the component doesn't need. I'd be more in favour of destructuring on top and passing what is needed to each. It sounds like it'd also be more efficient. What do you guys think?

Reply:
Yeah I ended up saying exactly the same thing actually, on my PR: https://github.com/yldio/yld.io/pull/237#issuecomment-498209125

I guess the thing I was pointing out here was about us having a consistent approach. @omonk any thoughts?

Reply:
It's more of a preference to be honest. passing props as `{...data}` doesn't have any difference in the resolution of data in JS. My reason for this approach is to localise any logic for a component to that component rather than having `open-source.js` contain a bunch of logic for all of the different sections/components. 

It's a static site so there is little business logic that we need to resolve as its mostly quite flat data.

Reply:
Cool. Thanks for the feedback Ollie. Let's park this for now then.